frontend/src/components/common/icon/styles.module.scss

```diff
@@ -4,3 +4,11 @@
   font-size: inherit;
   fill: currentcolor;
 }
+
+.delete {
+  fill: var(--typography-text-gray-200);
+}
+
+.delete:hover {
+  fill: var(--typography-text-100-inverse);
+}
```

Comment:
I think it's not a good idea to add this styles in icon component

Reply:
agreed. everything should be by `className` prop